### DISCUSSION TITLE
Single Logout

### DISCUSSION THREAD

**Discussion by dattito** [goauthentik/authentik#2495] [Q&A]
Saying I have authentik on "auth.domain.tld" and my website on "app.domain.tld". If I logout on "app.domain.tld", I also want to logout from "auth.domain.tld", because otherwise it isn't a real logout.

The only way I found is to logout from "app.domain.tld" and then redirect the user to the logout-flow page of authentik, because then they get "really" logged out. After that, they get redirected to the normal login site, after which they enter their authentik dashboard. But because I want to "hide" the other authentik sides from them because I just have one app, I want to redirect them back to my website "app.domain.tld" after they logged in again.

I think you can say that I want to redirect the user after he passed the logout-flow. Is this possible? Or is there a better way?

**Comment by BeryJu** [comment]
Logout in general is a topic in which authentik is currently lacking, I'm working on implementing proper SAML SLO and OIDC Logout support, after that's in you'll be able to use a flow for logouts instead of the currently fixed page. This will allow admins to fully configure what happens on logout.

**Reply by Fuseteam** [reply]
Is this currently still in the works?

**Reply by GD-Dal** [reply]
Still isn't in place as far as I can see

**Reply by Fuseteam** [reply]
i noticed that, i was more asking if there is work in progress on it or if it has stalled
